addons/search-sprites/userscript.js

```diff
@@ -1,3 +1,12 @@
+/* (remove this before commiting!!!)
+How i'll go about fixing this:
+  - Get the OG list of sprites from the redux store
+  - Loop through it to see which sprites belong to which folder
+  - When user searches for open folder, current system works fine
+  - When user searches for closed folder, bring up the folder
+  - user can open the folder without messing up search results
+*/
+
```

Comment:
You didn't remove this lol
```suggestion
```

Reply:
lol, I thought I removed it with commit 1c1d62c30132923006659d98af318b888f456317. Can you check again?

Reply:
Yea, you did remove them.